templates/zerver/help/set-notifications-for-a-single-stream.md

```diff
@@ -0,0 +1,20 @@
+# Set notifications for a single stream
+
+If you would like to have different notification settings for different streams,
+ you can set notifications for a single stream.
+
+1. To set notifications for a single stream, find the **Streams**
+ list located on the left sidebar.
+
+2. On the **Streams** list, hover over the stream that you want to set
+ notifications for to reveal a {!down-chevron.md!}.
+
+3. Click on the {!down-chevron.md!} and select the **Stream settings**
```

Comment:
You have to split this sentence into two as the expansion of **{!down-chevron.md!}** ends with a full stop.